fitbenchmarking/cli/main.py

```diff
@@ -122,10 +121,8 @@ def get_parser():
         nargs="+",
         type=str,
         default=[],
-        help=(
-            "Select what type of algorithm is used within a specific "
-            "software."
-        ),
+        help="Select what type of algorithm is used within a "
+        "specific software.",
```

Comment:
I think this is undoing some changes from the ruff PR?

Reply:
I think it might be an issue with how I rebased. I rebase on VS code and I tried to correct some. But I will put this back to what it was too.
<img width="1467" alt="image" src="https://github.com/user-attachments/assets/be05528f-3a2a-4fb8-9e4f-8640529c9dc2">